configuration/configuration.go

```diff
@@ -892,6 +906,14 @@ const (
 	minimumDeploymentsHTTPTimeout   = 1
 	defaultDeploymentsHTTPTimeout   = 30
 
+	// as of now deployments and codebase service is integrated in wit, but
+	// going forward this will change
+	// TODO: @surajssd : change the default values of `defaultDeploymentsServiceURL`
+	// and `defaultCodebaseServiceURL` to appropriate defaults, pointing to
+	// respective services.
+	defaultDeploymentsServiceURL = "http://wit:8080"
```

Comment:
In dsaas/rh-idev , `http://wit` gets us to the wit service, right ?

Reply:
done!